### PR TITLE
Harald

### PR DESCRIPTION
**Pull request by JorjMcKie**

### CODE REVIEW

**Comment by JorjMcKie** [comment]
This PR is still incomplete:
* Documentation changes to `Page.get_drawings()`/`.get_cdrawings()` already reflect changes caused by #2011. However, tests / confirmation by the client are still outstanding, so the corresponding code changes are not included here.
* I cannot confirm whether some required changes (made by Robin Watts) have been applied to MuPDF:
    - flags "embed" and "never_embed" for fonts have been implemented completely yet?
    - changes to XREF table maintenance for incremental updates have been applied to avoid mixed old-style and XREF stream updates?

File: docs/page.rst [hunk 1]
```diff
@@ -974,22 +976,27 @@ In a nutshell, this is what you can do with PyMuPDF:
...
 
       :arg int xref: the :data:`xref` of the image.
 
-      This is a **global replacement:** the image will disappear wherever the old one has been displayed throughout the file.
+      This is a **global "deletion":** the image will disappear wherever the old one has been displayed throughout the file.
+
+      If you inspect / extract a page's images by methods like :meth:`Page.get_images`, :meth:`Page.get_image_info` or :meth:`Page.get_text`, the replacing "dummy" image will be detected like so `(45, 47, 1, 1, 8, 'DeviceGray', '', 'Im1', 'FlateDecode')` and also seem to "cover" the same boundary box on the page.
    
    
+>>>>>>> Stashed changes
```
Comment (jamie-lemon) [inline docs/page.rst:999]: This looks wrong to me?

Comment (JorjMcKie) [reply docs/page.rst:999]: yes, it is wrong, I therefore changed the state to draft.

File: docs/page.rst [hunk 2]
```diff
@@ -974,22 +976,27 @@ In a nutshell, this is what you can do with PyMuPDF:
       This is a **global replacement:** the new image will also be shown wherever the old one has been displayed throughout the file.
 
       This method mainly exists for technical purposes. Typical uses include replacing large images by smaller versions, like a lower resolution, graylevel instead of colored, etc., or changing transparency.
+
+      The page is not "aware" of this replacement and will treat the new image like it did the old one, so for example the same bbox will be used.
```
Comment (jamie-lemon) [inline docs/page.rst:980]: bbox -> `bbox`` or -> boundary box

File: docs/page.rst (hunk 1 above)
Comment (jamie-lemon) [reply docs/page.rst:999]: aha - okay - I checked over all the RST stuff, and that was more or less all I found - but I'll look again once this comes out of "Draft".

**Pull request closed without merging**